packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -352,15 +361,19 @@ export class GCTelemetryTracker {
 				return;
 			}
 			this.loggedUnreferencedEvents.add(uniqueEventId);
+			deletedNodeIds.push(nodeId);
+		}
+
+		if (deletedNodeIds.length > 0) {
 			logger.sendTelemetryEvent({
-				eventName: "GCObjectDeleted",
-				id: nodeId,
-				type: nodeType,
-				age: currentReferenceTimestampMs - nodeStateTracker.unreferencedTimestampMs,
-				timeout: this.configs.sweepTimeoutMs,
-				completedGCRuns,
-				lastSummaryTime,
-				...this.createContainerMetadata,
+				eventName: "GC_SweepReadyObjects_Delete",
+				details: JSON.stringify({
+					id: JSON.stringify(deletedNodeIds),
```

Comment:
The `id` does not need to be JSON.stringified since the outer object is. I did that so that I can validate the id via MockLogger whose APIs accept ITelemetryBaseEvent types where the properties (id in this case) cannot be an array.